Sources/SnowManLogic/ViewLogic/MockIO.swift

```diff
@@ -0,0 +1,26 @@
+public class MockIO: Interactable {
+	public var stubbedUserInput: [String]!
+	public var displayCalled: Bool
+	public var getPrintedOutputStream: String!
+	var counter: Int
+
+	public init() {
+		displayCalled = false
+		counter = 0
+	}
+
+	public func getUserInput() -> String {
+		let value = stubbedUserInput[counter]
+		counter += 1
+		return value
+	}
+
+	public func display(stubbedPrintedOutput: String) {
+		displayCalled = true
+	}
+
+	public func stubPrintedOutput(getPrintedOutputStream: String) {
```

Comment:
this doesn't make sense. you don't stub the output, you want to make sure display is receiveing the correct output from the view. So your display function should store the string passed to it, and you should assert it received the right string. You should get rid of this `stubPrintedOutput` method all together